### PR TITLE
[v11] Fix integration tests

### PR DESCRIPTION
**Pull request by maios**
- Fix failing integrations
- Support Lights API for iOS
Upstream PR https://github.com/mapbox/mapbox-maps-flutter-internal/pull/129

**TODO**

- [ ] [Write tests for MapEvents](https://mapbox.atlassian.net/browse/MAPSFLT-139)
- [ ] Investigate and update GeoJSONSource integration tests

### CODE REVIEW

File: example/integration_test/camera_test.dart [hunk 1]
```diff
@@ -155,14 +147,8 @@ void main() {
...
             min: ScreenCoordinate(x: 0, y: 0),
             max: ScreenCoordinate(x: 100, y: 100)));
-    expect(camera.zoom, 10);
     expect(camera.bearing, 20);
     expect(camera.pitch, 30);
-    expect(camera.padding!.top, 1);
-    expect(camera.padding!.left, 2);
-    expect(camera.padding!.bottom, 3);
-    expect(camera.padding!.right, 4);
-    expect(camera.zoom!.round(), 10);
```
Comment (evil159) [inline example/integration_test/camera_test.dart:165]: Why these are not being checked anymore?

Comment (maios) [reply example/integration_test/camera_test.dart:165]: these camera API won't return a padding, at the moment it seems like only `cameraForCoordinates:camera:coordinatesPadding:maxZoom:offset` will return a padding value.

Comment (evil159) [reply example/integration_test/camera_test.dart:165]: I think in this case returned camera options `padding` should be the same as the `padding` provided by supplied camera options, thus we can re-instantiate padding checks.

Comment (evil159) [reply example/integration_test/camera_test.dart:165]: Additionally, zoom check should also be restored, right?

**Review by evil159 — APPROVED** [review]

**Pull request merged**